### DISCUSSION TITLE
Erase Screen Firmware

### DISCUSSION THREAD

**Discussion by AdamWP** [Desuuuu/DGUS-reloaded-Klipper#36] [General]
I'm having issues with Klipper and want to revert to older firmware but I can't seem to update my screen firmware successfully. When I flash the screen firmware with Creality or TinyMachines firmware the process looks like it's worked but when I reboot the Klipper logo still shows up. The new expected home screen eventually appears but won't work. It seems as though the DGUS reloaded firmware is still somewhat present. Is there a way to completely remove it?

**Comment by Desuuuu** [comment]
There's a bug caused by the bin file names IIRC. Try renaming the `13_*.bin` file from your new firmware to `13_touch.bin` and the `14_*.bin` file to `14_variable.bin` before flashing it again.

You can also try to replace the `T5UID1.CFG` from your new firmware with [this one](https://github.com/Desuuuu/DGUS-reloaded-Klipper/tree/master/extras/flash_reset) to erase the memory.

**Reply by AdamWP** [reply]
Thanks for the help. None of that worked though.

**Reply by Desuuuu** [reply]
https://github.com/Desuuuu/DGUS-reloaded-Klipper/wiki/Flashing-the-firmware

Your micro SD card might be improperly formatted (or incompatible).

**Reply by AdamWP** [reply]
The screen appears to flash when I use it though. Screen turns blue, files get transferred, and new screen files show up after the klipper spash screen on printer re-start. It's the same SD card I used to flash your firmware with. Would that still happen with an improperly formatted SD card?

**Reply by Desuuuu** [reply]
Yes, the screen reports 0 files flashed.

**Reply by AdamWP** [reply]
Looks like my SD card is good. Time for a screen upgrade.
